Examples/VimeoUpload-iOS-OldUpload/VimeoUpload-iOS-OldUpload/AppDelegate.swift

```diff
@@ -59,7 +59,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate
         self.window?.rootViewController = tabBarController
         self.window?.makeKeyAndVisible()
         
-        self.requestCameraRollAccessIfNecessary()
+        // Adding this here as work-around to stop system notifications from fuxing with
```

Comment:
Ehhh, I think maybe instead we could set the Host Application for this test target to None: 

<img width="1048" alt="screen shot 2017-01-24 at 11 51 17 am" src="https://cloud.githubusercontent.com/assets/19800/22257258/7f114c1e-e22b-11e6-910c-e48136a7dbb2.png">

But you'll still have the same problem for UI Tests target. 

Sooooooo since neither of these test targets are actually testing anything, and since this is for the old VimeoUpload stuff, maybe we just get rid of the placeholder tests and the old test targets? Any objection to that @mikew-personal?